python/graphstorm/model/lp_gnn.py

```diff
@@ -145,4 +145,11 @@ def lp_mini_batch_predict(model, emb, loader, device):
                     scores[canonical_etype].append(s)
                 else:
                     scores[canonical_etype] = [s]
-    return scores
+        # calculate ranking. etype is ignored in calculating ranking and
+        # global mrr. User can develop its own per etype MRR evaluator
+        rankings = []
```

Comment:
Can we return separated ranking tensors for different etypes?
Previous users only need to modify the evaluator to develop a per etype MRR evaluator. But now they have to modify the lp_mini_batch_predict, which is more complex.